.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 .vscode/
-npm-debug.log.*
\ No newline at end of file
+npm-debug.log.*
+yarn.lock
```

Comment:
@sinabakh it's better to keep `yarn.lock` according to this:

https://yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control